### ISSUE TITLE
Unable to rename spreadsheet tab in one of my notes

### ISSUE THREAD

**Issue by robfdavis** [standardnotes/forum#3431]
**Describe the bug**
I am working in an already created note and I converted to a spreadsheet. 
_Sheet1_ is current name of the only Sheet tab. 
When I double click mouse to rename the _Sheet1_, the _Sheet1_ tab name momentarily shows an editable name field with cursor at end of the current name _Sheet1_ but then quickly closes, not allowing me to type and my spreadsheet syncs. 
I add a second Sheet. It is named _Sheet2_. I double click mouse to rename _Sheet2_ and I am able to do so. 
If I move _Sheet1_ to be after _Sheet2_, I can then rename _Sheet1_. But _Sheet2_ is now unable to be renamed. 
If I create a brand new note and convert to a spreadsheet, this issue above does not occur. 

**To Reproduce**
1. Go to new note, change to Spreadsheet
2. Double Click on Sheet1 name 
3. Cursor flashes at end of name as thought I could edit, but then goes back to Sheet1 - cannot type to rename. 

**Expected behavior**
The ability to rename a Sheet

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Mac 14.1.2 (23B92)
 - Standard Notes App - version (3.183.22) 
 - Version 3.183.22

**Additional context**
I am not sure if there is just a bug in this note. But when I duplicated it, the same thing happens as described above.

**Comment by effieeee** [comment]
Hi @robfdavis, I'm afraid I haven't been able to reproduce this so far, but have you tried reinstalling the app on your device as an initial step? Also, do you experience the same issue while renaming the spreadsheet on the web app? (app.standardnotes.com)

**Comment by robfdavis** [comment]
Hi @effieeee, thank you very much for your message. I just finished testing out the reinstall on my Mac as well checked on the web app. Each time the same issue occurs where the one tab can't be renamed. So I also tried to duplicate the note, and interestingly, the same problem exists there. But I created a new note with no content and converted to a spreadsheet and I didn't have any trouble renaming the tab. I also created a new note, added some text, and converted to spreadsheet and I didn't have any trouble renaming the tab. I'm thinking it is just something off with that individual note. I'll just make a new note and work from there. I really appreciate your help trying to reproduce. I'll go ahead and close with my comment. Have a good evening!
